sdl-lang/src/test/scala/io/smartdatalake/meta/configexporter/DynamicTestTransformer.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2024 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.smartdatalake.meta.configexporter
+
+import io.smartdatalake.workflow.action.spark.customlogic.CustomDfsTransformer
+import io.smartdatalake.workflow.action.spark.transformer.Test
+import org.apache.spark.sql.{DataFrame, Dataset, SparkSession}
+
+class DynamicTestTransformer extends CustomDfsTransformer {
```

Comment:
Can you add a comment what this code is testing? I'm not sure I understand. Is it an end 2 end test for your feature? If yes, please indicate it. It can be confusing that you test your new feature in the tests of an unrelated feature.

In generel in my opinion it would be nice to have a complete example of a Test using your new feature so that it can serve as an implementation example for developpers. With complete example I mean one package where you clearly see the class used, the config file and sdlb being called end 2 end to show you the complete picture looks like.

Reply:
This is just a CustomDfsTransformer implementation to check that parameters are exported into the documentation with ConfigJsonExorter. It is used in ConfigJsonExporterTest.
I'll add some scala doc.
I will also add an end2end config test, thats a good idea.